packages/react-day-picker/src/components/Month/Month.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { useDayPicker, useNavigation } from '../../contexts';
+import { Table } from '../Table';
+
+/** The props for the [[Month]] component. */
+export interface MonthProps {
+  displayIndex: number;
+  displayMonth: Date;
```

Comment:
Seems a bit weird to pass in both of these and then also query for `displayMonths`...perhaps we only pass in displayMonth or displayIndex, and we compute the other based on displayMonths?

Reply:
I'd prefer we keep `displayMonth` and remove `displayIndex`, then get the index from the hoo. 

Still I have no idea how to write the test: I think I should mock the contex. I'd keep it like it is for this PR: being internal component shouldn't affect too much. Feel free to give a look to see if we can remove one prop.